octavia_proxy/tests/unit/api/drivers/elbv3/test_driver.py

```diff
@@ -15,8 +15,8 @@
 
 import requests
 from keystoneauth1 import adapter
-from otcextensions.sdk.vlb.v3 import load_balancer, listener, pool,\
-    member
+from otcextensions.sdk.vlb.v3 import load_balancer, listener, pool, \
+    member, health_monitor
```

Comment:
`\` for new line can be replaced with imports surrounded in parenthesis (`from ... import (...)`)